tests/testthat/test-score_forecasts.R

```diff
@@ -448,8 +484,60 @@ test_that("wis is correct, median only", {
   expect_equal(actual$wis, expected)
 })
 
+test_that("wis is correct, median only, use_median_as_point FALSE", {
```

Comment:
are we testing that "wis is correct" when we are expecting an error? please clarify/align expected behavior and name of test.